app/controllers/verify/step_controller.rb

```diff
@@ -8,5 +8,25 @@ class StepController < ApplicationController
     before_action :confirm_idv_attempts_allowed
 
     helper_method :step
+
+    private
+
+    def step_name
+      raise NotImplementedError, 'must implement step_name method'
+    end
+
+    def increment_step_attempts(increments = 1)
```

Comment:
we don't seem to ever call this with anything other than 1, can we remove this argument?